book/website/collaboration/research-infrastructure-roles.md

```diff
@@ -1,4 +1,14 @@
 (cl-research-infrastructure-roles)=
+
+```{figure} ../../figures/researchers-environment.*
+---
+height: 500px
+name: researchers-environment
+alt: The image is a complex amalgamation of people places and things. At the top of the image is a header that states "RESEARCHERS'" and at the bottom is a footer stating "ENVIRONMENT". The image contains the following. In the top left is a person in a construction crane carrying a series of ones and zeroes wearing a hard-hat looking down. They are labeled "Research Software Engineer". In the bottom left, we see an older man with a cane scratching his head looking at a math formula sketched onto the side of a building. Next to him is two pink fluffy rams. One is trapped insight of a light bulb. I hope they manage to escape... This scene is labeled "Research Application Managers". In the top right is a woman in a pink shirt sketching ones and zeroes onto the side of a bridge who is labeled "Data Stewards". In the bottom right is a series of cylinders with little stick figures climbing up ladders to a light bulb that is labeled "Useful Projects". They are being looked upon by a slightly amused looking man with short dark hair.
```

Comment:
Maybe the woman should be named "Data Steward" since it's only one (although I realise the text says "Data stewards"...)